README.md

```diff
@@ -45,16 +69,16 @@ To install the libraries necessary, run the following command:
 yarn
 ```
 
-### Millicast Tokens
+### Millicast (RTS) Tokens
```

Comment:
Do we still need to keep `Millicast` here?

Reply:
Na I'll remove it